### ISSUE TITLE
File settings

### ISSUE THREAD

**Issue by Mandarancio** [fabiocolacio/Marker#123]
I think it would  be nice to have some setting stored per file.
In such way that the editor remember the *view mode* used for the file and maybe *the last editing/scroll position*.

**Comment by fabiocolacio** [comment]
Yes that sounds like a good idea. I particularly like the idea of keeping the scroll position. This is a feature that *Gedit* has. It seems that *Gedit* stores your cursor position in the document when you close the document, and the cursor is still in the same place when it is re-opened.
